.evergreen.yml

```diff
@@ -57,13 +29,12 @@ functions:
           set -o errexit
           . venv/bin/activate
 
-
           if [ "${is_patch}" = "true" ]; then
             # Do not deploy on patches.
             exit 0
           fi
 
-          scripts/deploy.sh ${pypi_user} ${pypi_password}
+          poetry publish --username ${pypi_user} --password ${pypi_password}
```

Comment:
Does this prevent overwriting existing versions?

Reply:
It does not. I haven't been able to find a good way to do that with any of the python publishing tools without manually writing something around them.